charts/prompt-keda/values.yaml

```diff
@@ -66,9 +66,9 @@ s3:
   endpointUrl: ""
   # -- If set, define environment variables with S3 credentials from this application's Vault secret.
   auth_env: false
-  # -- If set, get a S3 credential file from this application's Vault secret.
+  # -- If set, get a S3 credential file from this application's Vault secret. Must be true if `aws_profile` is set.
   cred_file_auth: true
-  # -- If set, specify a S3 credential profile.
+  # -- If set, specify a S3 credential profile and `cred_file_auth` must be true. If empty and `auth_env`=`false`, the `default` profile is used.
```

Comment:
Should this be "If empty and `cred_file_auth = true`"? I wouldn't expect the profile to interact with `auth_env` either way.

Reply:
I suggest dividing the sentences one per line, like is done for e.g. `raw_microservice`.

Reply:
1. `aws_profile` is set, `cred_file_auth` must be `true`. The profile is used no matter `auth_env` is `true` or `false`  (the two S3 cred env vars are set or not; they are ignored anyway). 
2. `aws_profile` is not set. `cred_file_auth` can be either `true` or `false`. If `auth_env` is `true`, the env vars are used. No profile is used.
3. `aws_profile` is not set and `auth_env` is `false`, the `default` profile is used and expected, otherwise failure.  
  
I was trying to cover the (2) scenario, so didn't want to say "the `default` profile is used when `aws_profile` is empty".  I guess the wording is still confusing?